### PR TITLE
add npins

### PR DESCRIPTION
**Pull request by phanirithvij**

### CODE REVIEW

File: README.md
Comment (cyntheticfox) [inline README.md:162]:
```suggestion
* [npins](https://github.com/andir/npins) - A simple tool for handling different types of dependencies in a Nix project. It is inspired by and comparable to Niv.
```

To make the linter happy

**Review by cyntheticfox — CHANGES_REQUESTED** [review]

**Review by cyntheticfox — APPROVED** [review]
looks good now. provided the linters pass, will merge

**Comment by phanirithvij** [comment]
Can the lint step be made to run without maintainers approval?

**Comment by Mic92** [comment]
It's a repository setting. Since I don't expect too much spam on this repository probably a good idea.

**Comment by cyntheticfox** [comment]
Afaict, you can't turn it off entirely, but I did loosen it to only require approval for first-time contributes with new github accounts (from all first-time contributors)

**Pull request merged**
